recipes/mecab-ko/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set name = "mecab-ko" %}
+{% set version = "0.996" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/erip/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0a0fb40058a08f97179fb506a8166830d73533e016c5054e7048b5d39e17c2a1
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+about:
+  home: https://github.com/erip/mecab-ko
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: BSD
```

Comment:
```suggestion
  license_file:
    - COPYING
    - GPL
    - LGPL
    - BSD
```

Reply:
nice - this is what I was a bit concerned with. didn't realize you could list multiples!